addons/l10n_ec/data/account.account.template.csv

```diff
@@ -113,7 +117,11 @@ ec2105010104,2105010104,Otras provisiones a favor de los empleados,TRUE,liabilit
 ec21050102,21050102,Otras provisiones locales,TRUE,liability_current,l10n_ec_ifrs
 ec210502,210502,Provisiones del exterior,TRUE,liability_current,l10n_ec_ifrs
 ec2106,2106,Porcion corriente de obligaciones emitidas,FALSE,liability_current,l10n_ec_ifrs
-ec21070101,21070101,Impuesto a la renta retenido a empleados,TRUE,liability_current,l10n_ec_ifrs
```

Comment:
please avoid unnecessary change here of the xml_id

Reply:
@stevTresCloud @pepetreshere